### ISSUE TITLE
adapt-contrib-narrative: accessibility issue

### ISSUE THREAD

**Issue by canstudios-nicolaw** [adaptlearning/adapt_framework#2417]
There are some accessibility issues when using this component in mobile view.

### Your environment
* 0.7.0 AT/ 4.0.1 FW
* NVDA screen reader with Chrome/Firefox/IE11

### Steps to reproduce
The strapline is not read out by screen readers on mobile. The screen reader reads out items in the following order (which seems strange):
- Item title
- Alt text for the image
- Item body

### Expected behaviour
The strapline should be read out by screen readers, the items should be read out in a logical order.

I realise that this could be a low priority issue due to it being a mobile-only accessibility issue.

**Comment by oliverfoster** [comment]
Yup, this is pretty much expected. Narrative mobile view needs some serious work. I was going to wait until after v5 release to tackle this but I'm happy if you're willing to get it working?

**Comment by canstudios-nicolaw** [comment]
I can't make any promises right now I'm afraid, I was just raising this for the record rather than expecting anyone to pick it up immediately. We've got an internal discussion about priorities tomorrow morning so I will bring this up and see how much work we would be willing to put into this component in the near future. We have got a lot of interest in helping to make more components accessible at the moment (it's just driven by which ones our content team use the most).

**Comment by oliverfoster** [comment]
We've concentrated on desktop view only at the moment. Which leaves the small subset of mobile adaptive components which aren't currently accessible at that size.
